src/app/components/AddressBox/__tests__/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import copy from 'copy-to-clipboard'
+import { screen } from '@testing-library/dom'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { AddressBox } from '../index'
+
+jest.mock('copy-to-clipboard')
+
+const PLACEHOLDER = 'placeholder'
+const TEST_ADDRESS = 'oasis1qqurxkgavtcjjytneumeclx59ds3avjaqg7ftqph'
```

Comment:
we use camelcase convention

Reply:
For both normal variables and constants?

Reply:
(we do use uppercase for some constants)